include/Metabox/Base.php

```diff
@@ -301,4 +301,41 @@ protected function render_result( $posts_moved ) {
 			'updated'
 		);
 	}
+
+	protected function render_roles_dropdown( $name = '', $hide_no_role_option = false ) {
```

Comment:
Let's create two new abstract classes called `PostMetabox` and `UserMetabox` which will inherit `Base` Metabox class.

All common functions that are specific to Post and Users can be moved to these classes. Let's also remove the `BM_Util_Helper` class.

Reply:
> Let's create two new abstract classes called PostMetabox and UserMetabox which will inherit Base Metabox class.

Classes created. I've named them as `PostBase` and `UserBase` as we have the name folder, under which these files are present, as `Metabox`